tests/rules/test_float_values.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (C) 2022 the yamllint contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from tests.common import RuleTestCase
+
+
+class FloatValuesTestCase(RuleTestCase):
+    rule_id = 'float-values'
+
+    def test_disabled(self):
+        conf = (
+            'float-values: disable\n'
+            'new-line-at-end-of-file: disable\n'
+            'document-start: disable\n'
+        )
+        self.check('angle: 0.0', conf)
+        self.check('angle: .NaN', conf)
+        self.check('angle: .INF', conf)
+        self.check('angle: .1', conf)
+        self.check('angle: 10e-6', conf)
```

Comment:
Custom tags and anchors/aliases are missing here.

Reply:
Added